lerna.json

```diff
@@ -0,0 +1,4 @@
+{
+  "lerna": "2.0.0-beta.7",
+  "version": "1.1.35"
```

Comment:
The current version is 1.1.41, but we should probably do a major version bump with this change!